web/admin/src/CurrentContent.js

```diff
@@ -214,11 +215,11 @@ export default class CurrentContent extends PureComponent {
 
 function iconFor(c) {
   switch (c.type) {
-    case 'html':
-    case 'text': return TextIcon
-    case 'web': return WebIcon
-    case 'survey': return PageIcon
-    default: return null
+    case 'survey': return <img className="current-content__icon" src={PageIcon} alt="survey"/>
+    case 'text': return <img className="current-content__icon" src={TextIcon} alt="text"/>
+    case 'html': return <img className="current-content__icon-html" src={HTMLIcon} alt="html"/>
+    case 'web': return <img className="current-content__icon" src={WebIcon} alt="web"/>
+    default: return <div/>
```

Comment:
I think we had trouble with this being a <div/> before.  I think `null` (render nothing) is a better default than an empty div that may mess with layout.